src/sass/app_2/modules/trading.scss

```diff
@@ -144,6 +144,17 @@
                 span.field-info {
                     margin: 0 0 0 auto;
                 }
+                div:nth-child(2) {
+                    margin-left: 4px;
+                }
+            }
+            .payout-wrapper, .profit-wrapper, .returns-wrapper {
```

Comment:
This makes a very long selector when compiled, and also harder to maintain. (Adding every single line of info there, will need to add a new class here only to apply exactly the same styles)

Do we really need unique classes for each one of them?

Reply:
The different classes had different styles until I decided to group them, probably best to make them share a single class for now.